src/lib/viewers/image/ImageViewer.js

```diff
@@ -116,6 +116,23 @@ class ImageViewer extends ImageBaseViewer {
         this.setScale(this.imageEl.offsetwidth, this.imageEl.offsetHeight);
     }
 
+    /**
+     * Getter for modifyWidthInsteadOfHeight which will be used in zoom
+     * and will determine if the width or the height needs to be modified
+     *
+     * @private
+     * @param {number} [width] - The width in px
+     * @param {number} [height] - The height in px
+     * @return {boolean} true if the width will be modified instead of height
+     */
+    getModifyWidthInsteadOfHeight(width, height) {
```

Comment:
Why not go with something like `isLandscape()`, since we're essentially checking whether or not the width is greater than the height? Also, we could reduce this to `return width >= height`. Were there any other reasons behind using aspect ratio?

Note: We may already have an `isLandscape()` method hiding somewhere

Reply:
Good point, the CR was just moving around code but I don't see the aspect ratio being used anywhere